analyzer/src/main/kotlin/PackageJsonUtils.kt

```diff
@@ -79,7 +84,15 @@ internal class PackageJsonUtils {
         }
 
         private fun isYarnWorkspaceRoot(definitionFile: File): Boolean {
-            return definitionFile.readValue<ObjectNode>()["workspaces"] != null
+            return try {
```

Comment:
The return type and the return could be replace by '='

Reply:
Done in a separate commit.